cmd/crane/cmd/mutate.go

```diff
@@ -100,8 +110,21 @@ func NewCmdMutate(options *[]crane.Option) *cobra.Command {
 			fmt.Println(r.Context().Digest(digest.String()))
 		},
 	}
+	mutateCmd.Flags().StringSliceVarP(&anntns, "annotation", "a", nil, "New annotations to add")
 	mutateCmd.Flags().StringSliceVarP(&lbls, "label", "l", nil, "New labels to add")
 	mutateCmd.Flags().StringVar(&entrypoint, "entrypoint", "", "New entrypoing to set")
 	mutateCmd.Flags().StringVarP(&newRef, "tag", "t", "", "New tag to apply to mutated image. If not provided, push by digest to the original image repository.")
 	return mutateCmd
 }
+
+func splitKeyValuePairsInFormOfAnEqualSign(kvPairs []string) (map[string]string, error) {
```

Comment:
This is a bit wordy. How about just `splitKeyVals` and a doc comment?

Reply:
done, thank you so much 🙏